main.go

```diff
@@ -224,37 +226,31 @@ func Unpin(b *hb.Bot, actor, path string) {
 	clusterPinUnpin(b, actor, path, "", false)
 }
 
+// StatusCluster gets cluster status of cid with given path.
 func StatusCluster(b *hb.Bot, actor, path string) {
 	ctx := context.Background()
-	// pick random cluster
-	i := rand.Intn(len(clusters))
-	cluster := clusters[i]
-	addr := clusterAddrs[i]
 
-	c, err := resolveCid(path, shs[i], shsUrls[i])
+	c, err := resolveCid(path, lbShell)
 	if err != nil {
 		botMsg(actor, fmt.Sprintf("could not resolve cid: %s", err))
 		return
 	}
 
-	st, err := cluster.Status(ctx, c, false)
+	st, err := lbClient.Status(ctx, c, false)
 	if err != nil {
-		botMsg(actor, fmt.Sprintf("%s: error obtaining pin status: %s", addr, err))
+		botMsg(actor, fmt.Sprintf("error obtaining pin status: %s", err))
```

Comment:
needs to keep reporting which cluster peer failed.

Reply:
A failure here indicates that all peers, that were tries, failed. Writing all peers would bloat the logs. 
- In case of failover we know that peers upto `retries` count in clients array have failed. So, logging failed peers here don't provide any extra information.
- In case of RoundRobin, we don't have such was know know tried peers since scheduled among multiple requests. So there logging failed peers provide us extra information. 

In general, an error in lb-client is indicative of problem in the cluster rather than individual clients. The err returned here is how last tried peer failed, though this does not tell us why exactly rest of the peers failed, it is likely why rest of the peers failed. Probably it will be useful if the error tells why all peers failed, map[peer.ID]error, or stringified form of that

Reply:
Yeah ok you're right, sorry I did not think of this